pdctl/command/region_command.go

```diff
@@ -101,28 +100,14 @@ func showRegionWithTableCommandFunc(cmd *cobra.Command, args []string) {
 		fmt.Println("Error: unknown format")
 		return
 	}
-
-	client, err := getClient()
-	if err != nil {
-		fmt.Println("Error: ", err)
-		return
-	}
-	region, leader, err := client.GetRegion(context.Background(), key)
+	prefix := fmt.Sprintf(regionKeyPrefix, key)
```

Comment:
the function `PathEscape` is new add in go 1.8. It is enough to deal with the key firstly with the protobuf format.